app/services/types_generator.rb

```diff
@@ -64,7 +64,7 @@ def build_genres
     # add the top level genre mapping (i.e. top level work type with no subtype)
     type_genres = types_to_genres.dig(work_type, 'type') || []
 
-    all_genres = type_genres + subtype_genres
+    all_genres = (type_genres + subtype_genres).uniq
```

Comment:
Per Arcadia in https://github.com/sul-dlss/happy-heron/issues/1254#issuecomment-790935330, we should not have duplicate genres if the same one is coming from both the top level mapping and then sub-type.  We will try and avoid duplicating in the .yml mappings, but this ensures it.

Reply:
Would you mind adding that to a comment in the code here?

Reply:
Would you mind adding that to a comment in the code here?

Reply:
Sure thing, added.